app/models/task.rb

```diff
@@ -0,0 +1,15 @@
+class Task < ActiveRecord::Base
+  def date_view
+    @date_completed = created_at.strftime("%m/%d/%Y")
+    return @date_completed
```

Comment:
If this method is returning this variable, this should probably be a local variable instead of an instance variable